### PR TITLE
Go to functionality

### PR DESCRIPTION
**Pull request by xion6136**

### CODE REVIEW

File: ClassySchedule/src/main/java/users/DeleteFacultyFromDatabaseController.java [hunk 1]
```diff
@@ -178,18 +194,23 @@ private String getLastName() {
...
-        // Iterate through the faculty table
-        JSONArray faculty = DatabaseStatic.getData("faculty");
-        for (Object jsonObject: faculty) {
-            JSONObject job = (JSONObject)jsonObject;
-            job.put("prev_load", String.valueOf( job.get("prev_load")));
-            // If matching selected faculty id then set result equal to previous course load and break
-            if (job.get("faculty_id").equals(selectedFaculty)) {
-                result = (String) job.get("prev_load");
-                break;
+        if (!(facultyID.getValue() == (null))) {
```
Comment (dtbielejeski) [inline ClassySchedule/src/main/java/users/DeleteFacultyFromDatabaseController.java]: don't need the parantheses around null

File: ClassySchedule/src/main/java/users/DeleteFacultyFromDatabaseController.java [hunk 2]
```diff
@@ -178,18 +194,23 @@ private String getLastName() {
...
+                JSONObject job = (JSONObject) jsonObject;
+                job.put("prev_load", String.valueOf(job.get("prev_load")));
+                // If matching selected faculty id then set result equal to previous course load and break
+                if (job.get("faculty_id").equals(selectedFaculty)) {
+                    result = (String) job.get("prev_load");
+                    break;
+                }
             }
+        } else {
+            result = "null";
```
Comment (dtbielejeski) [inline ClassySchedule/src/main/java/users/DeleteFacultyFromDatabaseController.java]: is this supposed to null without the quotes?

File: ClassySchedule/src/main/java/users/DeleteFacultyFromDatabaseController.java [hunk 3]
```diff
@@ -201,17 +222,21 @@ private String getPrevCourseLoad() {
...
+                JSONObject job = (JSONObject) jsonObject;
+                job.put("curr_load", String.valueOf(job.get("curr_load")));
+                // If matching selected faculty id then set result equal to current course load and break
+                if (job.get("faculty_id").equals(selectedFaculty)) {
+                    result = (String) job.get("curr_load");
+                    break;
+                }
             }
+        } else {
+            result = "null";
```
Comment (dtbielejeski) [inline ClassySchedule/src/main/java/users/DeleteFacultyFromDatabaseController.java]: same as above

File: ClassySchedule/src/main/java/users/DeleteFacultyFromDatabaseController.java [hunk 4]
```diff
@@ -201,17 +222,21 @@ private String getPrevCourseLoad() {
...
-        // Iterate through the faculty table
-        JSONArray faculty = DatabaseStatic.getData("faculty");
-        for (Object jsonObject: faculty) {
-            JSONObject job = (JSONObject)jsonObject;
-            job.put("curr_load", String.valueOf( job.get("curr_load")));
-            // If matching selected faculty id then set result equal to current course load and break
-            if (job.get("faculty_id").equals(selectedFaculty)) {
-                result = (String) job.get("curr_load");
-                break;
+        if (!(facultyID.getValue() == (null))) {
```
Comment (dtbielejeski) [inline ClassySchedule/src/main/java/users/DeleteFacultyFromDatabaseController.java]: same as above

**Review by dtbielejeski — CHANGES_REQUESTED** [review]

File: ClassySchedule/src/main/java/users/DeleteFacultyFromDatabaseController.java (hunk 4 above)
Comment (xion6136) [reply ClassySchedule/src/main/java/users/DeleteFacultyFromDatabaseController.java]: Fixed!

**Review by dtbielejeski — APPROVED** [review]

**Review by ZackSchuster600 — APPROVED** [review]
Nice code bro

**Pull request merged**